wordpress-mobile/WordPress-iOS
# Add word count label to editor

Added a word count function to UITextView.
  When editing/ creating post the word count label appears on HTML or Rich Text style when then number of words greater that 0

Fixes #10176 

To test:
* New Post *
- Create a new post 
- Type something in title
- Type something in text view. Notice that the word count label appears after the first word  
- Continue typing and notice the word count increases with every new word
- remove all content words and notice the label disappears when there are no words. 

* Editing post *
- Edit an existing post
- Notice that the word count label appears if the post had content
- Continue editing and notice the word count increases with every new word

* Same functionality should work for HTML and for Rich text.

![word_count](https://user-images.githubusercontent.com/1335657/46389688-50e77c80-c688-11e8-9083-ca67b1725bc7.gif)
